ledger/ledgercore/statedelta.go

```diff
@@ -234,23 +285,28 @@ func (ad NewAccountDeltas) ModifiedAccounts() []basics.Address {
 }
 
 // MergeAccounts applies other accounts into this StateDelta accounts
+// Used in tests only
```

Comment:
If it's used for testing only within the same package, consider moving it to a _test.go file.
That would also exclude it from code coverage.

Reply:
This is a mistake, it actually used in cow.go. Removed